frontend/src/model/BackendConnection.ts

```diff
@@ -22,19 +22,24 @@ export interface DbCommandItem {
     universe: number
 }
 
-export interface ActiveFixtureList {
+export interface ActiveFixtureItem {
     name: string,
     checked: number
 }
 
 export interface FaderPageSelect {
     startAddress: number,
-    quantity: number
+    quantity: number,
+    universe: number
```

Comment:
Isnt universe a complex type you saving in your db instead of a simple number? Or do you mean something else here?